pkg/prompt/prompt.go

```diff
@@ -108,3 +119,11 @@ func InputTypeError(prompterApp WalletPrompterInterface) error {
 
 	return fmt.Errorf(InputTypeErr)
 }
+
+func UserChoiceError(prompterApp WalletPrompterInterface) error {
+	fmt.Println("invalid user choice input")
+	prompterApp.EmitEvent(walletapp.PromptResultEvent,
+		walletapp.EventData{Success: false, CodeMessage: UserChoiceErr})
```

Comment:
I had put all the code messages in the utils package

```suggestion
		walletapp.EventData{Success: false, CodeMessage: utils.ErrUserChoice})
```